pkg/sbom/scanner/scanner_test.go

```diff
@@ -105,8 +105,8 @@ func TestRetryLogic_Error(t *testing.T) {
 
 	// Set up the configuration as the default one is too slow
 	cfg := config.NewConfig("datadog", "DD", strings.NewReplacer(".", "_"))
-	cfg.Set("sbom.scan_queue.base_backoff", "1s", model.SourceAgentRuntime)
-	cfg.Set("sbom.scan_queue.max_backoff", "3s", model.SourceAgentRuntime)
+	cfg.Set("sbom.scan_queue.base_backoff", "200ms", model.SourceAgentRuntime)
+	cfg.Set("sbom.scan_queue.max_backoff", "600s", model.SourceAgentRuntime)
```

Comment:
Should this be ms ?

Reply:
Yes, good catch 👍 In any case it should run for at most 400ms.